PHP/factorial.php

```diff
@@ -0,0 +1,17 @@
+<?php
```

Comment:
Needs ?> tag as well at the end of the file

Reply:
@utkarsh-shekhar I can add it no problem, but the ?> tag is optional.

http://php.net/basic-syntax.instruction-separation

Reply:
Apologies. I just found out that the end tag is optional. Approved and merging this PR